src/studio/src/designer/frontend/shared/navigation/main-header/Header.test.tsx

```diff
@@ -1,20 +1,18 @@
 import React from 'react';
-import {
-  createTheme as createThemeV4,
-  ThemeProvider as ThemeProviderV4,
-} from '@material-ui/core/styles';
+import { createTheme, ThemeProvider } from '@mui/material';
+
 import { render as rtlRender, screen } from '@testing-library/react';
 
+import type { IHeaderContext } from './Header';
 import {
+  getOrgNameById,
   Header,
   HeaderContext,
-  getOrgNameById,
   SelectedContextType,
 } from './Header';
-import type { IHeaderContext } from './Header';
 import altinnTheme from '../../theme/altinnStudioTheme';
 
-const themeV4 = createThemeV4(altinnTheme);
+const themeV4 = createTheme(altinnTheme);
```

Comment:
No longer uniquely themeV4 aka same as themeV5...
```suggestion
const theme = createTheme(altinnTheme);
```

Reply:
Denne er fikset, kommer i neste commit.